### PR TITLE
feat: Check pore volume for all element types

### PR DESCRIPTION
**Pull request by paveltomin**
also check that default aperture > 0

### CODE REVIEW

**Review by CusiniM — APPROVED** [review]

**Comment by CusiniM** [comment]
This PR needs more codeowners approval.

**Review by rrsettgast — APPROVED** [review]

**Review by jhuang2601 — APPROVED** [review]

**Pull request merged**
